superbuild/projects_modules/VTK.cmake

```diff
@@ -66,15 +66,14 @@ set(cmake_args
   -DCMAKE_MACOSX_RPATH:BOOL=OFF
   -DCMAKE_SHARED_LINKER_FLAGS=${${ep}_shared_linker_flags}
   -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS_${ep}}
-  -DBUILD_TESTING=OFF
-  -DBUILD_DOCUMENTATION=OFF
-  -DBUILD_EXAMPLES=OFF
+  -DVTK_BUILD_TESTING=OFF
+  -DVTK_BUILD_DOCUMENTATION=OFF
+  -DVTK_BUILD_EXAMPLES=OFF
   -DVTK_RENDERING_BACKEND=OpenGL2
-  -DVTK_Group_Qt=ON
-  -DModule_vtkGUISupportQtOpenGL=ON
-  -DModule_vtkRenderingOSPRay:BOOL=${USE_OSPRay}
   -DVTK_QT_VERSION=5
-  -DVTK_USE_OGGTHEORA_ENCODER:BOOL=ON # OGV Export
+  -DVTK_MODULE_ENABLE_VTK_GUISupportQt=YES
+  -DVTK_MODULE_ENABLE_VTK_RenderingQt=YES
```

Comment:
Attention au mélange YES et ON

Reply:
(seen, it's the new cmake system, it's not boolean anymore here)